src/adapters/utils/networkUtils.js

```diff
@@ -157,10 +158,11 @@ const processAxiosResponse = (clientResponse) => {
     };
   }
   // success(2xx) axios response
-  const { data, status } = clientResponse.response;
+  const { data, status, headers } = clientResponse.response;
   return {
```

Comment:
Will there be any effect for integrations using `processAxiosResponse` due to the this change in structure ?

Reply:
It will not be breaking change. It will return `headers` when someone uses `processAxiosResponse`.

Reply:
`processAxiosResponse` is being used directly in CDK flow for few integrations - like Gladly

Reply:
Verified and modified